docs/topic-guides/blendable-web-apps/view-model-attaching.md

```diff
@@ -156,6 +156,24 @@ In the example above, converter passes handler parameters to token parameters, a
 
 Often it's needed to trigger attachment on a specific URI. To achieve this, first converter should return non-null string array on certain parameters.
 
+If both attached URIs define the same number of parameters then you will get the same set of parameters in both handlers, but if they define a different number of parameters then you will get only those for which wildcards are existing starting from the first.
+
+ ```csharp
+Handle.GET("/op1/{?}", (string first) =>
+{
+    return first;
+});
+Handle.GET("/op2/{?}/{?}", (string first, string second) =>
+{
+    return first + " and " + second;
+});
+
+Blender.MapUri("/op1/{?}", "token1");
+Blender.MapUri("/op2/{?}/{?}", "token1");
+
+Response resp = Self.GET("/op2/first/second");
```

Comment:
Please add a comment in this line saying what will happen. I guess it should be:

Reply:
Its simpler. Current URI matcher selects the most concrete URI handler possible, among all choices. Its not related to number of parameters. Obviously, for `/op2/first/second`, more concrete handler is `/op2/{?}/{?}` and not `/op1/{?}`

Reply:
Hi @alemoi, could you please take a look at my recently added changes (which uses your explanation)?

Reply:
Yes, that's better. Approved!